src/interface/src/app/plan/create-scenarios/create-scenarios.component.ts

```diff
@@ -209,16 +212,16 @@ export class CreateScenariosComponent implements OnInit, OnDestroy {
 
   private loadConfig(): void {
     this.planService.getProject(this.scenarioConfigId!).subscribe((config) => {
-      const estimatedCost = this.formGroups[1].get('budgetForm.estimatedCost');
-      const maxCost = this.formGroups[1].get('budgetForm.maxCost');
-      const maxArea = this.formGroups[1].get('physicalConstraintForm.maxArea');
-      const excludeDistance = this.formGroups[1].get(
+      const estimatedCost = this.formGroups[2].get('budgetForm.estimatedCost');
```

Comment:
similarly here, it would improve readability if we do something like
```
const someGroupOfForms = this.formGroups[2];
const otherGroupOfForms = this.formGroups[3];
const estimatedCost = someGroupOfForms.get('budgetForm.estimatedCost');
```